### ISSUE TITLE
1.01 is recognized as a virus by Microsoft Defender and is automatically deleted

### ISSUE THREAD

**Issue by Jaylinx** [unexpectedpanda/retool#147]
**Describe the bug**
After downloading 1.01 version, a warning from Microsoft Defender is shown. Then, file is deleted.

**Operating system**
- Windows 11

**Comment by unexpectedpanda** [comment]
My first false positive! This means I've made it, right? 😅

This one could be difficult to track down -- I'm running Win 10, where Defender reports no viruses found. Not a lot changed between Retool 1.00 and 1.01, which makes me wonder if Defender on Win 11 is picking up something in the way this version of Retool was compiled.

I've updated my version of PyInstaller (which creates the exe from the Python code), and recompiled. If this file fixes the issue, I'll replace the 1.01 archive with this version:

https://unexpectedpanda.github.io/files/retool-1.01a-win-x86-64.zip

If not, the only solution really is to add an exception to Defender to tell it to trust the file.
